homeassistant/components/androidtv/media_player.py

```diff
@@ -216,6 +217,7 @@ def __init__(
     ):
         """Initialize the Android TV / Fire TV device."""
         self.aftv = aftv
+        self._attr_device_class = MediaPlayerDeviceClass.TV
```

Comment:
You could move this above `__init__` and set it as `_attr_device_class = MediaPlayerDeviceClass.TV` since it never changes